# 0 (disable) 1 (enable)
int32 power

# 0 (off) 1 (on)
int32 motor

# current voltage in mv
int32 voltage
